pkgs/servers/sql/mariadb/connector-c/default.nix

```diff
@@ -39,10 +40,6 @@ stdenv.mkDerivation {
     ln -sv mariadb_config $out/bin/mysql_config
     ln -sv mariadb $out/lib/mysql
     ln -sv mariadb $out/include/mysql
-    ln -sv libmariadbclient.a $out/lib/mariadb/libmysqlclient.a
-    ln -sv libmariadbclient.a $out/lib/mariadb/libmysqlclient_r.a
-    ln -sv libmariadb.so $out/lib/mariadb/libmysqlclient.so
-    ln -sv libmariadb.so $out/lib/mariadb/libmysqlclient_r.so
```

Comment:
Why are you removing these, I'm pretty sure some software required them?

Reply:
"-DWITH_MYSQLCOMPAT=ON" - this option automatically creates symlinks.

Reply:
ls -lah ./result/lib/mariadb
```
итого 459K
dr-xr-xr-x 3 root root   10 янв  1  1970 .
dr-xr-xr-x 4 root root    5 янв  1  1970 ..
-r--r--r-- 1 root root 511K янв  1  1970 libmariadbclient.a
lrwxrwxrwx 1 root root   15 янв  1  1970 libmariadb.so -> libmariadb.so.3
-r-xr-xr-x 1 root root 396K янв  1  1970 libmariadb.so.3
lrwxrwxrwx 1 root root   18 янв  1  1970 libmysqlclient.a -> libmariadbclient.a
lrwxrwxrwx 1 root root   18 янв  1  1970 libmysqlclient_r.a -> libmariadbclient.a
lrwxrwxrwx 1 root root   15 янв  1  1970 libmysqlclient_r.so -> libmariadb.so.3
lrwxrwxrwx 1 root root   15 янв  1  1970 libmysqlclient.so -> libmariadb.so.3
dr-xr-xr-x 2 root root    9 янв  1  1970 plugin
```